velox/docs/functions/spark/decimal.rst

```diff
@@ -0,0 +1,8 @@
+=================
+Decimal functions
+=================
+
+.. spark:function:: unscaled_value(x) -> bigint
+
+    Return the unscaled bigint value of a short decimal ``x``, assuming it fits in a bigint.
```

Comment:
> assuming it fits in a bigint.

This statement is redundant since any short decimal fits into a bigint. Let's remove it.